src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 15px;
+  width: 200px;
+  border-radius: 5px;
+  background-color: #fff;
+  color: $color-main;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    padding-inline: 3px;
+    margin-bottom: 36px;
+    & img {
+      width: 100%;
+      height: 134px;
+      background-size: cover;
+    }
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    color: $color-bold;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    margin: 0;
+    line-height: 14px;
+    margin-bottom: 16px;
```

Comment:
There is a duplicated `margin-bottom` property. The first one on line 31 is overridden by the second one on line 33. Remove the unnecessary declaration to keep the code clean.